Backend/prisma/schema.prisma

```diff
@@ -10,9 +10,13 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model users {
+model user  {
```

Comment:
I know it might by annoying. But conventionally , types/models must have same naming as classes and types. 
CamelCase (start with capital letter)

Now let me explain more, We have two different things.
1. User Type/Entity. (this concept derived from our application)
2. users collection/table (this concepts derived from Database world)

So what people agreed about is that: 
1. table names should. be 
     -- snake_case (lower case)
     -- Plural form 
2. Entity/Model types (any thing that is application related)
    -- CameCase (start with upper case letter)
    -- Singular form

In Prisma you can achieve this, define the model name and table name differently.
Look at this https://www.prisma.io/docs/orm/prisma-schema/data-model/database-mapping#map-collection--table-names

Since this is not a blocking change I am going to approve the PR, but please work on fixing it.

Reply:
I put it as User after you told me that, but after I did that, I had a problem in the service section when I made a query on the database, as I had to put the query as .User, but Prisma did not accept to take it in this form as .user, so the command did not work for me except  Returning the table name to user, I will try to look into it further tomorrow.